app/controllers/schools_controller.rb

```diff
@@ -8,23 +8,25 @@ def index
   end
 
   # GET /schools/1 or /schools/1.json
-  def show; end
+  def show
+  end
```

Comment:
I like the single lines aesthetically, but I'm not going to fight the standardrb defaults over it

Reply:
:D  

Now you're feeling that ["unconfigurable configuration"](https://github.com/standardrb/standard#:~:text=and%20provides%20an-,unconfigurable%20configuration,-to%20all%20of) vibe.